core/field_dropdown.js

```diff
@@ -421,15 +429,19 @@ Blockly.FieldDropdown.prototype.isOptionListDynamic = function() {
 
 /**
  * Return a list of the options for this dropdown.
- * @return {!Array.<!Array>} Array of option tuples:
+ * @param {boolean=} opt_useCache For dynamic options, whether or not to use the
+ *     cached options or to re-generate them.
+ * @return {!Array.<!Array>} A non-empty array of option tuples:
  *     (human-readable text or image, language-neutral name).
  * @throws If generated options are incorrectly structured.
```

Comment:
nit: Could we add the type here? @throws {Type}

Reply:
Done.